README.md

```diff
@@ -17,6 +15,52 @@ This will launch your browser, where the demo page will be displayed (at http://
 
```

Comment:
Hmm. Running `npm run start` did not, in  fact, launch my browser.

And Webpack told me that the project is running at http://127.0.0.1:8080/ not at http://localhost:3000